# Keshav Motwani

<img align="right" src="keshav_headshot.jpg" width="273.6" height="410.4" alt="Keshav Motwani headshot">
 

I am a third-year PhD student in biostatistics at the University of Washington, funded by an NSF Graduate Research Fellowship.

I am currently working with Ali Shojaie and Eardi Lila on methods for high-dimensional statistical learning and inference for dependent data settings, with applications to neuroimaging data. In my first project I developed a  efficient method to estimate covariance matrices in multivariate mixed models. I previously worked with Daniela Witten on inference after using a machine learning model to predict the outcome variable of interest and data thinning, an alternative to sample splitting in unsupervised settings. 

I did my undergraduate degree at the University of Florida in mathematics and statistics. I worked with [Aaron Molstad](https://ajmolstad.github.io/) and [Rhonda Bacher](https://www.rhondabacher.com/) on high-dimensional multinomial regression with multiresolution/hierarchical categories. I also worked with [Todd Brusko](https://bruskolab.diabetes.ufl.edu/) and [Victor Greiff](https://greifflab.org/) on analyzing and developing software for immune receptor sequencing and single-cell genomics data in the context of type 1 diabetes. 

You can see some of my work on my [Google Scholar](https://scholar.google.com/citations?user=8AhTuSEAAAAJ&hl=en) profile and my [Github](https://github.com/keshav-motwani) page.

My CV is available [here](https://github.com/keshav-motwani/CV/blob/master/CV.pdf).

**Publications**

Methods
1. **Keshav Motwani** and Daniela Witten. Revisiting valid inference after prediction. _Journal of Machine Learning Research_,
2023 [Paper]([https://arxiv.org/abs/2306.13746](https://www.jmlr.org/papers/v24/23-0896.html)), [Code](https://github.com/keshav-motwani/PredictionBasedInference)
2. Ameer Dharamshi, Anna Neufeld, **Keshav Motwani**, Lucy L Gao, Daniela Witten,
and Jacob Bien. Generalized data thinning using sufficient statistics. _Journal of the American Statistical Association_, 2024 [Paper](https://arxiv.org/abs/2303.12931)
3. Aaron J Molstad and **Keshav Motwani**. Multiresolution categorical regression for
interpretable cell type annotation. _In press at Biometrics_, 2023 [Paper](https://arxiv.org/abs/2208.13857), [Software](https://github.com/ajmolstad/HierMultinom/)
4. **Keshav Motwani**, Rhonda Bacher, and Aaron J Molstad. Binned multinomial logistic regression for integrative cell type annotation. _In press at Annals of Applied Statistics_, 2023 [Paper](https://arxiv.org/abs/2111.12149), [Software](https://github.com/keshav-motwani/IBMR/)

Applied
1. Melanie R Shapiro, Xiaoru Dong, Daniel Perry, James M McNichols, Puchong Thirawatananond, Amanda L Posgai, Leeana Peters, **Keshav Motwani**, Richard Musca, Andrew Muir, and others. Human immune phenotyping reveals accelerated aging in type 1 diabetes. _JCI Insight_, 2023 [Paper](https://insight.jci.org/articles/view/170767)
2. Chakravarthi Kanduri, Milena Pavlovic, Lonneke Scheffer, **Keshav Motwani**, Maria Chernigovskaya, Victor Greiff, and Geir K Sandve. Profiling the baseline performance and limits of machine learning models for adaptive immune receptor repertoire classification. _GigaScience_, 11, 2022 [Paper](https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giac046/6593147)
3. Milena Pavlovic, Lonneke Scheffer, **Keshav Motwani**, Chakravarthi Kanduri, Rad- mila Kompova, Nikolay Vazov, Knut Waagan, Fabian LM Bernal, Alexandre Almeida Costa, Brian Corrie, and others. immuneML: an ecosystem for machine learning analysis of adaptive immune receptor repertoires. _Nature Machine Intelligence_, 2021 [Paper](https://www.nature.com/articles/s42256-021-00413-z), [Software](https://immuneml.uio.no/)
4. Peter S Linsley, Fariba Barahmand-pour Whitman, Elisa Balmas, Hannah A DeBerg, Kaitlin J Flynn, Alex K Hu, Mario G Rosasco, Janice Chen, Colin ORourke, Elisavet Serti, Vivian H Gersuk, **Keshav Motwani**, and others. Autoreactive T cell receptors with shared germline-like α chains in type 1 diabetes. _JCI Insight_, 2021 [Paper](https://insight.jci.org/articles/view/151349/)
5. **Keshav Motwani**, Leeana D Peters, Willem H Vliegen, Ahmed Gomaa El-Sayed, Howard R Seay, M Cecilia Lopez, Henry V Baker, Amanda L Posgai, Maigan A Brusko, Daniel J Perry, and others. Human regulatory T cells from umbilical cord blood display increased repertoire diversity and lineage stability relative to adult peripheral blood. _Frontiers in Immunology_, 11:611, 2020 [Paper](https://www.frontiersin.org/articles/10.3389/fimmu.2020.00611/full), [Code](https://github.com/keshav-motwani/tregPaper)
6. Emmi-Leena Ihantola, Henna Ilmonen, Anssi Kailaanmaki, Marja Rytkonen-Nissinen, Aurelien Azam, Bernard Maillere, Cecilia S Lindestam Arlehamn, Alessandro Sette, **Keshav Motwani**, Howard R Seay, and others. Characterization of proinsulin T cell epitopes restricted by type 1 diabetes–associated HLA class II molecules. _The Journal of Immunology_, 204(9):2349–2359, 2020 [Paper](https://journals.aai.org/jimmunol/article/204/9/2349/107608/Characterization-of-Proinsulin-T-Cell-Epitopes)
7. Mohsen Khosravi-Maharlooei, Aleksandar Obradovic, Aditya Misra, **Keshav Motwani**, Markus Holzl, Howard R Seay, Susan DeWolf, Grace Nauman, Nichole Danzl, Haowei Li, and others. Cross-reactive public TCR sequences undergo positive selection in the human thymic repertoire. _The Journal of Clinical Investigation_, 129(6):2446–2462, 2019 [Paper](https://www.jci.org/articles/view/124358)
